db/re/skill_db.yml

```diff
@@ -16920,6 +16933,31 @@ Body:
     HitCount: 1
     SplashArea: 13
     Duration1: 3000
+  - Id: 723
+    Name: NPC_STORMGUST2
+    Description: Storm Gust 2
+    MaxLevel: 3
+    Type: Magic
+    TargetType: Ground
+    Flags:
+      IsNpc: true
+      ShowScale: true
+    Range: 9
+    Hit: Single
+    HitCount: 1
+    Element: Water
+    Knockback: 2
```

Comment:
based on playing in KRO this skill doesn't have knock back. (use by Ominous Freezer)